lib/ansible/modules/system/sysctl.py

```diff
@@ -211,12 +211,17 @@ def _values_is_equal(self, a, b):
     def _parse_value(self, value):
         if value is None:
             return ''
-        elif isinstance(value, bool):
+        if isinstance(value, bool):
             if value:
                 return '1'
             else:
                 return '0'
-        elif isinstance(value, basestring):
+        # Check for python 2.x basestring which unifies (unicode, str) support
+        try:
+            basestring
+        except NameError:
+            basestring = str
+        if isinstance(value, basestring):
```

Comment:
We should be utilizing `ansible.module.utils.six.string_types` instead of declaring `basestring` in py3.

Reply:
That part of basic.py is there for backwards compatibility.  Using six is the current way we do things.